### PR TITLE
Rewire recent modules to the standard, fix wrongly edited title

### PR DESCRIPTION
**Pull request by adamkvitek**
Moving modules to subfolders, renaming to satisfy recommendation from [Docusaurus docs](https://docusaurus.io/docs/markdown-features/react#importing-markdown).

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: src/components/concept/_ci-ignoring-files.mdx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 By default `semgrep ci` skips files and directories such as `tests/`, `node_modules/`, and `vendor/`. It uses the default `.semgrepignore` file which you can find in the [Semgrep GitHub repository](https://github.com/returntocorp/semgrep/blob/develop/.semgrepignore). This default is used when no explicit `.semgrepignore` file is found in the root of your repository.
 
-You can copy and commit the default `.semgrepignore` file to the **root of your repository** and extend it with your own entries or write one from scratch. If Semgrep detects a `.semgrepignore` file within your repository, it does not append entries from the default `.semgrepignore` file.
+Copy and commit the default `.semgrepignore` file to the **root of your repository** and extend it with your own entries or write your `.semgrepignore` file from scratch. If Semgrep detects a `.semgrepignore` file within your repository, it does not append entries from the default `.semgrepignore` file.
```
Comment (s-santillan) [inline src/components/concept/_ci-ignoring-files.mdx:3]: I wonder why we have to say "copy and commit" - this is not something the user has to do. .semgrepignore will function even when the user doesn't do it.

Comment (s-santillan) [reply src/components/concept/_ci-ignoring-files.mdx:3]: (What I mean to say is, it's optional.)

Comment (adamkvitek) [reply src/components/concept/_ci-ignoring-files.mdx:3]: @s-santillan I agree. Feel free to change this or suggest and edit... so should this stay as:
```suggestion
Optional: Copy and commit the default `.semgrepignore` file to the **root of your repository** and extend it with your own entries or write your `.semgrepignore` file from scratch. If Semgrep detects a `.semgrepignore` file within your repository, it does not append entries from the default `.semgrepignore` file.
```

Comment (adamkvitek) [reply src/components/concept/_ci-ignoring-files.mdx:3]: To express this in a more TW way...

**Review by s-santillan — APPROVED** [review]

**Comment by s-santillan** [comment]
My previous PR [705](https://github.com/returntocorp/semgrep-docs/pull/705/files) overwrote the **correct** names to the old ones. This PR attempted to fix. I'll take over for now as those were my errors. Fortunately I haven't pulled the latest main, so it's just a case of copy pasting my current files to this PR.

**Pull request merged**